README.md

```diff
@@ -0,0 +1,22 @@
+# World Portals
+
+A module to add teleportation portals. Portals currently occupy a block and transport any entities that enter the occupied block to the portals destination world position.
```

Comment:
"A module to add teleportation portals."
This feels.. abrupt. Rephrasing it to "This module adds ..." makes it feel more natural.

"... to the portals destination ..."
You missed an apostrophe here. (Not trying to be a grammar nazi, the lack of apostrophe changes the meaning, throwing off the reader)